packages/cli/src/endo.js

```diff
@@ -370,7 +371,7 @@ export const main = async rawArgs => {
         storeJson,
         storeJsonStdin,
         storeBigInt,
-        name,
+        name: parsePetNamePath(name),
```

Comment:
Really ought to be done in `store.js` to keep `endo.js` light and loosely coupled.